code/modules/jobs/job_types/assistant.dm

```diff
@@ -21,6 +21,14 @@ Assistant
 
 	family_heirlooms = list(/obj/item/storage/toolbox/mechanical/old/heirloom, /obj/item/clothing/gloves/cut/heirloom)
 
+	mail_goodies = list(
+		/obj/effect/spawner/lootdrop/donkpockets = 10,
+		/obj/item/clothing/mask/gas = 10,
+		/obj/item/clothing/gloves/color/fyellow = 7,
+		/obj/item/choice_beacon/music = 5,
+		/obj/item/clothing/gloves/color/yellow/sprayon = 3
```

Comment:
I think you're looking for **/obj/item/toy/sprayoncan** here

Reply:
In this case, based on the prior comment thread, I replaced the insulated gloves at a weight of 10 with a weight of 7 for budget insuls, and 3 for spray-on insulated gloves.

Reply:
No I mean, these are the abstract nodrop gloves you're given when you spray your hands with the can instead of the actual spray you're supposed to use to apply them